x-pack/plugins/uptime/e2e/journeys/monitor_management.journey.ts

```diff
@@ -16,10 +17,10 @@ const basicMonitorDetails = {
   location: customLocation || 'US Central',
   schedule: '3',
 };
-const httpName = 'http monitor';
-const icmpName = 'icmp monitor';
-const tcpName = 'tcp monitor';
-const browserName = 'browser monitor';
+const httpName = `http monitor ${uuid.v4()}`;
+const icmpName = `icmp monitor ${uuid.v4()}`;
+const tcpName = `tcp monitor ${uuid.v4()}`;
+const browserName = `browser monitor ${uuid.v4()}`;
```

Comment:
If you don't do this you will risk conflicting monitor names when executing the runner multiple times against the same server if any failures occur.